app/javascript/mastodon/components/status.js

```diff
@@ -71,6 +72,7 @@ class Status extends ImmutablePureComponent {
     'account',
     'muted',
     'hidden',
+    'askReplyConfirmation',
```

Comment:
Thinking about it, I'm not too sure about the performance implications of this. Maybe the `mapDispatchToProps` thing should be rewritten instead to use `getState`.